spanner/go.mod

```diff
@@ -3,7 +3,7 @@ module cloud.google.com/go/spanner
 go 1.11
 
 require (
-	cloud.google.com/go v0.57.0
+	cloud.google.com/go v0.58.0
```

Comment:
Is this intentional?

Reply:
Yes. I made some changes in https://github.com/googleapis/google-cloud-go/blob/master/internal/testutil/trace.go, which are included in v.0.58.0.